test/test_va_api_query_vendor.cpp

```diff
@@ -26,28 +26,22 @@
 
 namespace VAAPI {
 
-// no addtl. fixture functionality needed... just a unique test name
 typedef VAAPIFixture VAAPIQueryVendor;
 
 TEST_F(VAAPIQueryVendor, Intel_i965_Vendor)
 {
-    const char* vendor = NULL;
-    std::string vendorString, findIntel("Intel i965 driver");
```

Comment:
`findIntel` should be declared as `const std::string`...

or you can just remove this variable and and pass `"Intel i965 driver"` directly to `vendor.find`

Reply:
...better yet, just declare and assign `vendor`  in one statement:

```
const std::string vendor(vaQueryVendorString(display));
```